src/app-airtable.js

```diff
@@ -83,6 +63,55 @@ const AIRTABLE_FIELDS = {
   COACHES: {},
 };
 
+// Function to transform Airtable records into Client entities
+function transformToClientEntities(records) {
+  return records.map((record) => {
+    const fields = record.fields;
+    // USER_IDS_SHARED_WITH that looks like an array in Airtable is a JSON string representation of an Array.
+    let parsedSharedWith;
+    try {
+      parsedSharedWith =
+        fields[AIRTABLE_FIELDS.CLIENTS.USER_IDS_SHARED_WITH] &&
+        fields[AIRTABLE_FIELDS.CLIENTS.USER_IDS_SHARED_WITH].length
+          ? JSON.parse(fields[AIRTABLE_FIELDS.CLIENTS.USER_IDS_SHARED_WITH][0])
```

Comment:
This was sort of working before, because string responds to `includes`, similarly to array, but it was definitely buggy.